src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -248,7 +248,7 @@ void VehicleIMU::Run()
 
 			// rotate sensor clip counts into vehicle body frame
 			const Vector3f clipping{_accel_calibration.getBoardRotation() *
-				Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
+						Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
```

Comment:
Can you revert this part? It's failing on CI, I think there are conflicting astyle versions.

Reply:
It wasn't a voluntary change. Somehow I had Astyle version 3.1 installed and it didn't let me commit without a `make format.` I now reverted to 2.06.